src/website/app/demos/Text/examples/color.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+import palette from '../../../../../library/colors';
+import { createStyledComponent } from '../../../../../library/styles';
+import { mineralTheme as theme } from '../../../../../library/themes';
+import Text from '../../../../../library/Text';
+
+const DemoPanel = createStyledComponent(
+  'div',
+  ({ backgroundColor, theme }) => ({
+    backgroundColor,
+    padding: theme.space_inset_sm
+  })
+);
+
+export default {
+  id: 'color',
+  title: 'Color',
+  description: `Text can be any color, but be sure to use a
+color/background-color combination with an
+[adequate contrast ratio](/color#guidelines-accessibility).`,
+  scope: { DemoPanel, palette, Text, theme },
+  source: `
+    <div>
+      <Text color={theme.color_text_success}>Success Text</Text>
+      <Text color={palette.magenta_60}>Magenta Text</Text>
```

Comment:
Suggest to rename `palette` to `colors` to match the folder name and perhaps make a better connection for the user.